src/test/unit/components/dropdown/Dropdown.test.tsx

```diff
@@ -0,0 +1,85 @@
+import { fireEvent, screen } from "@testing-library/react";
+import { renderWithProviders } from "../../../../test__utils/renderWithProvides";
+
+import { DROPDOWN_LINKS } from "../../../../components/Dropdown/DropdownConstants";
+
+import Dropdown from "../../../../components/Dropdown/Dropdown";
+
+describe("Dropdown", () => {
+  const setIsDropdownVisible = jest.fn();
+
+  test("render dropdown items", () => {
+    renderWithProviders(
+      <Dropdown
+        dropdownLinks={DROPDOWN_LINKS}
+        setIsDropdownVisible={setIsDropdownVisible}
+      />
+    );
+
+    expect(DROPDOWN_LINKS).toHaveLength(6);
+
+    const homeLink = screen.getByRole("link", { name: "Home" });
+    expect(homeLink).toBeInTheDocument();
+
+    const userStatusLink = screen.getByRole("link", { name: "Status" });
+    expect(userStatusLink).toBeInTheDocument();
+
+    const profileLink = screen.getByRole("link", { name: "Profile" });
+    expect(profileLink).toBeInTheDocument();
+
+    const taskLink = screen.getByRole("link", { name: "Tasks" });
+    expect(taskLink).toBeInTheDocument();
+
+    const identityLink = screen.getByRole("link", { name: "Identity" });
+    expect(identityLink).toBeInTheDocument();
+
+    const signoutButton = screen.getByRole("button", { name: "Sign out" });
+    expect(signoutButton).toBeInTheDocument();
```

Comment:
Do you think declaring all variable first, then testing for being in document would be more readable?